src/pages/komodo-defi-framework/api/v20/my_recent_swaps/index.mdx

```diff
@@ -0,0 +1,295 @@
+
+#### Response
+
+| Structure      | Type             | Description                                                                                                                                                                                            |
+| -------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| swaps          | array of objects | A list of standard [SwapStatus](/komodo-defi-framework/api/common_structures/swaps/#swap-status) objects                                                                                               |
+| from\_uuid     | string           | the from\_uuid that was set in the request; this value is null if nothing was set                                                                                                                      |
+| skipped        | number           | the number of skipped records (i.e. the position of `from_uuid` in the list + 1 or `(page_number - 1) * limit`; the value is 0 if `from_uuid` or `page_number` were not set or `page_number` is 1)     |
+| limit          | number           | the limit that was set in the request; note that the actual number of swaps can differ from the specified limit (e.g. on the last page)                                                                |
+| total          | number           | total number of swaps available with the selected filters                                                                                                                                              |
+| page\_number   | number           | the page\_number that was set in the request; if both `page_number` and `from_uuid` are not set in request it will default to `1`; if `from_uuid` is present in request this value will be always null |
+| total\_pages   | number           | total pages available with the selected filters and limit                                                                                                                                              |
+| found\_records | number           | the number of returned swaps                                                                                                                                                                           |
+
+#### 📌 Example
+
+#### Command
+
+<CodeGroup title="" tag="POST" label="my_recent_swaps" mm2MethodDecorate="true">
+  ```json
+  {
+    "mmrpc": "2.0",
+    "userpass": "RPC_UserP@SSW0RD",
+    "method": "my_recent_swaps",
+    "params": {
+      "filter": {
+        "my_coin": "BTC",
+        "other_coin": "ETH",
+        "from_timestamp": 1672531200,
+        "to_timestamp": 1704067200,
+        "from_uuid": null,
+        "limit": 5,
+        "page_number": 1
+      }
+    }
```

Comment:
This is how it should be with all structs flattened
```json
    "params": {
        "my_coin": "BTC",
        "other_coin": "ETH",
        "from_timestamp": 1672531200,
        "to_timestamp": 1704067200,
        "from_uuid": null,
        "limit": 5,
        "page_number": 1
    }
```

Reply:
fixed